src/components/ha-humidifier-state.js

```diff
@@ -0,0 +1,83 @@
+import { html } from "@polymer/polymer/lib/utils/html-tag";
+/* eslint-plugin-disable lit */
+import { PolymerElement } from "@polymer/polymer/polymer-element";
+import LocalizeMixin from "../mixins/localize-mixin";
+
+/*
+ * @appliesMixin LocalizeMixin
+ */
+class HaHumidifierState extends LocalizeMixin(PolymerElement) {
```

Comment:
We don't accept new `PolymerElement`s please convert this to `LitElement`.

Reply:
Done